lib/Styles/Common/_lighting.scss

```diff
@@ -91,3 +91,14 @@
         inset 0 1px 1px rgba(black, 0.05),
         inset 0 0 1px 1px rgba(black, 0.05);
 }
+
+@mixin lighting-protrude {
```

Comment:
Probably don't want this because, for example, the progressbar uses multiple background gradients. In the old stylesheet we had:

```
// Background for elements with outset style like headerbars, buttons, checkboxes, etc
%outset-background {
    background-image:
        linear-gradient(
            to bottom,
            #{'alpha(@highlight_color, 0.2)'},
            rgba(white, 0)
        );

    &:backdrop {
        background-image:
            linear-gradient(
                to bottom,
                #{'alpha(@highlight_color, 0.35)'},
                #{'alpha(@highlight_color, 0.3)'}
            );
    }
}
```